tests/integration/secretsmanager/test_secretsmanager.py

```diff
@@ -0,0 +1,1700 @@
+import json
+import os
+import uuid
+from datetime import datetime
+from typing import Dict, List, Optional
+
+import pytest
+import requests
+from py._code.code import ExceptionInfo
+
+from localstack.constants import TEST_AWS_ACCOUNT_ID
+from localstack.services.awslambda.lambda_utils import LAMBDA_RUNTIME_PYTHON36
+from localstack.utils import testutil
+from localstack.utils.aws import aws_stack
+from localstack.utils.strings import short_uid
+from localstack.utils.time import today_no_time
+from tests.integration.awslambda.test_lambda import TEST_LAMBDA_PYTHON_VERSION
+from tests.integration.secretsmanager.functions import lambda_rotate_secret
+
+RESOURCE_POLICY = {
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {"AWS": "arn:aws:iam::%s:root" % TEST_AWS_ACCOUNT_ID},
+            "Action": "secretsmanager:GetSecretValue",
+            "Resource": "*",
+        }
+    ],
+}
+
+THIS_FOLDER = os.path.dirname(os.path.realpath(__file__))
+TEST_LAMBDA_ROTATE_SECRET = os.path.join(THIS_FOLDER, "functions", "lambda_rotate_secret.py")
+
+
+class TestSecretsManager:
+    @pytest.fixture
+    def sm_client(self):
+        return aws_stack.create_external_boto_client("secretsmanager")
+
+    @pytest.mark.parametrize(
```

Comment:
Since the files have been moved, it's a bit hard to figure out the changes - can you point us at the new/modified tests that have been added?